.phpcs.xml.dist

```diff
@@ -49,7 +49,7 @@
 			<!-- Provide the plugin specific prefix for use with namespace and hook names. -->
 			<property name="prefixes" type="array">
 				<element value="JoostBlog\WP\Comment"/>
-				<element value="yoast_comment"/>
+				<element value="emiliaprojects_comment"/>
```

Comment:
Should we just stick to `comment_hacks`?

Reply:
Done